aspect/intellij_info_impl.bzl

```diff
@@ -704,6 +712,10 @@ def build_java_package_manifest(ctx, target, source_files, suffix):
         arguments = [args],
         mnemonic = "JavaPackageManifest",
         progress_message = "Parsing java package strings for " + str(target.label),
+        execution_requirements = {
```

Comment:
I think it would be great to also support multiplexed workers here.

Reply:
@keithl-stripe is this something that makes sense from your perspective?

Reply:
Sure, but for now that's a "nice to have" - I believe it would simply improve overall performance, but should not be a blocker for this PR.